website/project/model.py

```diff
@@ -389,6 +389,8 @@ class NodeLog(StoredObject):
     action = fields.StringField(index=True)
     params = fields.DictionaryField()
     should_hide = fields.BooleanField(default=False)
+    original_node = fields.ForeignField('node', index=True)
+    node = fields.ForeignField('node', index=True)
     __indices__ = [
         {
             'key_or_list': [
```

Comment:
It appears to tell mongo what to index on (as opposed to telling modularodm what to index on via the `index` kwarg in a `fields` constructor), and I haven't noticed any problems removing it locally.